Examples/Swift/OHHTTPStubsDemo.xcodeproj/project.pbxproj

```diff
@@ -328,6 +333,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.3;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
 				SWIFT_VERSION = 2.3;
```

Comment:
This is the project level settings which is overwritten by the target level settings. I'll change that and remove the target level one.